### ISSUE TITLE
Feel sad for planning to replace this library 

### ISSUE THREAD

**Issue by rizkiheryandi** [tymondesigns/jwt-auth#2152]
👋 tymondesigns/jwt-auth team,

I'm Rizki from [uasoft](https://github.com/uasoft-indonesia) and maintainer [badaso](https://github.com/uasoft-indonesia/badaso)

First of all, i would like to thanks for all this library maintainer and contrinbutor, it's a awsome project.

Because of that, we are using this in our low-code development platform called [badaso](https://github.com/uasoft-indonesia/badaso) and that's awesome !

But badaso is a long term project and need other LTS library project also

But until now, this library looks like a abandoned project, it's so sad and very unfortunate.

So we have plan to replace this with other good support token auth library from the official called [sanctum](https://github.com/laravel/sanctum).

I really hope there is a reply from the this library maintainer about the future this library.

So we can make better choices and plan

If badaso get good financial support, for sure we will be a sponsoe for this library

Thank you

**Comment by Messhias** [comment]
> 👋 tymondesigns/jwt-auth team,
> 
> I'm Rizki from [uasoft](https://github.com/uasoft-indonesia) and maintainer [badaso](https://github.com/uasoft-indonesia/badaso)
> 
> First of all, I would like to thank you for all this library maintainer and contributors, it's an awesome project.
> 
> Because of that, we are using this in our low-code development platform called [badaso](https://github.com/uasoft-indonesia/badaso) and that's awesome!
> 
> But badaso is a long term project and need other LTS library project also
> 
> But until now, this library looks like an abandoned project, it's so sad and very unfortunate.
> 
> So we have a plan to replace this with another good support token auth library from the official called [sanctum](https://github.com/laravel/sanctum).
> 
> I hope there is a reply from this library maintainer about the future of this library.
> 
> So we can make better choices and plan
> 
> If badaso get good financial support, for sure we will be a sponsor for this library
> 
> Thank you

Hi, I and @eschricker decide to work and do a separate project forked by the library and we're now working to get all the pull requests and transfer them to the new project.

You're welcome to contribute with us.

https://github.com/PHP-Open-Source-Saver/jwt-auth

**Comment by rizkiheryandi** [comment]
Good, thanks for the information @Messhias

**Comment by rizkiheryandi** [comment]
Yes sure, i will contribute and support the good library as much as i can @Messhias @eschricker

**Comment by rizkiheryandi** [comment]
Could you assign me as maintainer also ? @Messhias 

thank you

**Comment by Messhias** [comment]
Sure, we'll.



*Best Regards,*

*Fabio William Conceição*

*Remote: https://remote.com/fabiowilliam <https://remote.com/fabiowilliam>*
*Tel/Whastapp: +351 93 212 1477*
*Skype: fabioo.william.conceicao*
*LinkedIn:
https://www.linkedin.com/in/fabio-william-concei%C3%A7%C3%A3o-379b9823/
<https://www.linkedin.com/in/fabio-william-concei%C3%A7%C3%A3o-379b9823/>*
*Github: **https://github.com/Messhias/ <https://github.com/Messhias/>*
*Upwork: https://www.upwork.com/o/profiles/users/_~0126d10487b9843f68/
<https://www.upwork.com/o/profiles/users/_~0126d10487b9843f68/>*


Em qui., 30 de set. de 2021 às 07:45, Rizki Heryandi <
***@***.***> escreveu:

**Comment by Messhias** [comment]
https://github.com/PHP-Open-Source-Saver/jwt-auth

**Comment by rcerljenko** [comment]
https://github.com/rcerljenko/laravel-jwt

**Comment by bobbarn** [comment]
> 
I am also super appreciative of this project and fully understand that when you make such a big contribution to the community it is not easy to maintain forever. I will also need to migrate my project to Sanctum for the same reasons.
@[rizkiheryandi](https://github.com/rizkiheryandi) if you have run into any really obvious gotchas I would love to know. I have a sanctum branch I have been working on for a while :)


>
